[[null, "Traceback (most recent call last):\r\n  File \"C:\\Users\\aantr\\AppData\\Local\\Programs\\Python\\Python38\\lib\\py_compile.py\", line 144, in compile\r\n    code = loader.source_to_code(source_bytes, dfile or file,\r\n  File \"<frozen importlib._bootstrap_external>\", line 846, in source_to_code\r\n  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\r\n  File \"B:\\Sasha\\01_Sync\\Exercises\\Yandex\\\u2567\u0401\u044e\u0445\u044a\u0404\u221a\\Test system\\program_testing\\source_solution\\thread_2\\solution.py\", line 1\r\n    race laundry travel spike axis whale gun already eager spirit faith case\r\n         ^\r\nSyntaxError: invalid syntax\r\n\r\nDuring handling of the above exception, another exception occurred:\r\n\r\nTraceback (most recent call last):\r\n  File \"C:\\Users\\aantr\\AppData\\Local\\Programs\\Python\\Python38\\lib\\py_compile.py\", line 209, in main\r\n    compile(filename, doraise=True)\r\n  File \"C:\\Users\\aantr\\AppData\\Local\\Programs\\Python\\Python38\\lib\\py_compile.py\", line 150, in compile\r\n    raise py_exc\r\n__main__.PyCompileError:   File \"B:\\Sasha\\01_Sync\\Exercises\\Yandex\\\u2567\u0401\u044e\u0445\u044a\u0404\u221a\\Test system\\program_testing\\source_solution\\thread_2\\solution.py\", line 1\r\n    race laundry travel spike axis whale gun already eager spirit faith case\r\n         ^\r\nSyntaxError: invalid syntax\r\n\r\n\r\nDuring handling of the above exception, another exception occurred:\r\n\r\nTraceback (most recent call last):\r\n  File \"C:\\Users\\aantr\\AppData\\Local\\Programs\\Python\\Python38\\lib\\runpy.py\", line 193, in _run_module_as_main\r\n    return _run_code(code, main_globals, None,\r\n  File \"C:\\Users\\aantr\\AppData\\Local\\Programs\\Python\\Python38\\lib\\runpy.py\", line 86, in _run_code\r\n    exec(code, run_globals)\r\n  File \"C:\\Users\\aantr\\AppData\\Local\\Programs\\Python\\Python38\\lib\\py_compile.py\", line 218, in <module>\r\n    sys.exit(main())\r\n  File \"C:\\Users\\aantr\\AppData\\Local\\Programs\\Python\\Python38\\lib\\py_compile.py\", line 213, in main\r\n    if quiet < 2:\r\nNameError: name 'quiet' is not defined\r\n", null, null]]